### PR TITLE
Upgrade GDP Master to baseline 11 (MIRANDA)

### PR DESCRIPTION
**Pull request by chbae**
[GDP-281] Upgrade GDP Master to baseline 11 (MIRANDA)

### CODE REVIEW

**Comment by chbae** [comment]
This PR should be merged with https://github.com/GENIVI/meta-genivi-dev/pull/24

**Comment by tom--pollard** [comment]
I'd like to include a few more commits in this PR to fall under the same merge commit. Mainly the changes to readme + init.sh. @RobertAJMarshall once you have made your changes can you make it available to @chbae ?

**Comment by tom--pollard** [comment]
Re my last comment about a single merge commit, it is not that important ultimately.

**Comment by tom--pollard** [comment]
I believe dragonboard support should be merged after this PR, in the RC1 cycle. This commit will form the initial commit of the beta release branch, with single target qemux86-64

**Comment by tom--pollard** [comment]
Great +1 here

File: init.sh [hunk 1]
```diff
@@ -25,7 +25,17 @@ if test -z "$machine" ; then
    unset machine choice
    return
 else
-   if [[ "$machine" == "dragonboard-410c" ]] && test -z "$eula" ; then
+    if [[ "$machine" != "qemux86-64" ]] ; then
+       echo "However for the current GDP-11 beta release only the qemux86-64 board is supported."
+       echo "You can either "
+       echo " re-run with the qemux86-64  target"
```
Comment (chbae) [inline init.sh]: need to change. wait.

**Pull request merged**